### ISSUE TITLE
New activation toggles are not accessible

### ISSUE THREAD

**Issue by enejb** [Automattic/jetpack#2580]
Currently you can't tab into them and activate or deactivate the toggles. Which would make them keyboard accessible. 

![image](https://cloud.githubusercontent.com/assets/115071/9367486/735f57a4-4672-11e5-8411-b470f38fc972.png)

**Comment by jeffgolenski** [comment]
Good find @enejb 

I pulled over the toggle styling + html from wpcom. But obviously that isn't enough for accessibility. What did you guys do to make this work for plugin mgmt on wpcom? JavaScript event handlers (like 'onclick') on not focusable elements?

cc @beaulebens @MichaelArestad @dereksmart

**Comment by kraftbj** [comment]
@davidakennedy may have suggestions too.

**Comment by davidakennedy** [comment]
Hey all!

I spent a few minutes with this, and may be missing a few things because I'm not that familiar with the code. But I think I can point you all in the right direction to make these work better for everyone. :smile:

So the reason the toggles do not work with the keyboard is:

```html
<span class="form-toggle__switch"></span>
```

A `span` is not keyboard accessible by default. Only form inputs, links and buttons are. So a keyboard user or user with assistive technology couldn't use these.

I'm assuming your JavaScript is firing based on that switch. An easy fix would be to change it to a `button` element, since that will be focusable with no extra JavaScript. You may need to cancel out a few styles. You'd also want to make sure it has a `:focus` style that's visible so people know when they're focused on it.

The other interesting thing is that there is a checkbox, but it's `display: none;`, so keyboard users or assistive technology users can't get to it. But that may be there as a backup if JS doesn't fire?

I hope this helps – let me know if you need anything else.
